We end up getting linker errors about us not finding /lib/... and some
various other bits due to the value of --sysroot, which is set to
somewhere in ${S}.

This might not be the ideal patch (or solution), but for us, we want
it to remain untouched for LDFLAGS in any case.

diff '--color=auto' -ru rumpkernel-0_p20250111_p6/buildrump.sh/src/share/mk/bsd.own.mk rumpkernel-0_p20250111_p6-fix/buildrump.sh/src/share/mk/bsd.own.mk
--- rumpkernel-0_p20250111_p6/buildrump.sh/src/share/mk/bsd.own.mk	2026-03-24 23:05:49.473069851 -0400
+++ rumpkernel-0_p20250111_p6-fix/buildrump.sh/src/share/mk/bsd.own.mk	2026-03-24 23:05:32.816580907 -0400
@@ -434,23 +434,8 @@
 #
 DESTDIR?=
 
-# Don't append another copy of sysroot (coming from COMPATCPPFLAGS etc.)
-# because it confuses Coverity. Still we need to cov-configure specially
-# for each specific sysroot argument.
-# Also don't add a sysroot at all if a rumpkernel build.
-.if !defined(HOSTPROG) && !defined(HOSTLIB) && !defined(RUMPRUN)
-.  if ${DESTDIR} != ""
-.	if empty(CPPFLAGS:M*--sysroot=*)
-CPPFLAGS+=	--sysroot=${DESTDIR}
-.	endif
-LDFLAGS+=	--sysroot=${DESTDIR}
-.  else
-.	if empty(CPPFLAGS:M*--sysroot=*)
-CPPFLAGS+=	--sysroot=/
-.	endif
-LDFLAGS+=	--sysroot=/
-.  endif
-.endif
+CPPFLAGS?=
+LDFLAGS?=
 
 DBSYM=		${TOOLDIR}/bin/${MACHINE_GNU_PLATFORM}-dbsym
 ARM_ELF2AOUT=	${TOOLDIR}/bin/${_TOOL_PREFIX}arm-elf2aout
